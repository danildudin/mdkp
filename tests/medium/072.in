29 9
792 99 20 312 358 5 883 212 599 899 714 231 936 267 996 349 512 130 781 112 862 610 892 762 20 491 630 263 570 
54 257 430 117 658 205 26 61 535 850 240 302 901 107 827 74 746 107 676 332 916 670 430 318 679 472 381 461 6 8611
166 643 825 350 552 634 883 611 955 979 310 711 168 347 674 347 851 825 959 140 716 661 353 597 302 640 44 529 131 6430
359 210 419 559 296 310 217 59 450 927 3 476 568 314 54 605 701 283 822 698 937 5 362 211 617 721 748 267 443 8111
781 508 695 839 180 749 809 875 482 452 17 669 598 577 895 820 312 731 503 72 174 656 578 941 32 787 659 243 543 7207
71 726 689 764 396 712 377 34 888 986 854 97 1 508 694 573 880 170 925 712 555 678 836 361 618 200 109 571 857 8747
108 794 328 894 263 33 810 927 653 478 927 641 526 695 536 515 164 597 343 438 70 863 993 566 440 360 298 389 353 9097
525 68 997 815 784 909 393 259 128 495 747 765 961 282 352 568 49 978 423 119 126 249 898 642 652 231 414 556 582 7255
597 527 120 730 592 226 893 118 41 175 61 935 128 881 65 148 301 47 463 353 71 26 134 524 843 50 743 352 635 9355
883 582 785 768 87 365 255 209 656 240 718 575 460 193 638 4 742 91 524 696 280 961 739 47 992 56 708 338 661 5110
