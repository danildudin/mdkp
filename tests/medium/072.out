9155
12
0 4 6 9 10 12 14 20 22 23 25 28 
